unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarHUDController.cs

```diff
@@ -715,9 +716,9 @@ private void OpenClickedChat(string chatId, string previous)
             OpenChatList();
         else
         {
-            async UniTaskVoid OpenPrivateChatAsync(string chatId)
+            async UniTaskVoid OpenPrivateChatAsync(string chatId, CancellationToken cancellationToken = default)
```

Comment:
You introduced `CancelationToken` but it's always `None` as you don't provide it

Reply:
done